server/apps/distributed-app/docker-compose.yml

```diff
@@ -32,37 +37,47 @@ services:
       - discovery.type=single-node
       - DISABLE_INSTALL_DEMO_CONFIG=true
       - DISABLE_SECURITY_PLUGIN=true
+    networks:
+      james:
+        aliases:
+          - elasticsearch
 
   cassandra:
-    image: cassandra:3.11.10
+    image: cassandra:4.1.3
```

Comment:
These changes should be on the origin master already. Let's rebase on the latest origin master to avoid "staying with the past code".